hazelcast/src/main/java/com/hazelcast/instance/impl/HazelcastBootstrap.java

```diff
@@ -584,42 +628,55 @@ public void shutdown() {
             this.jobName = jobName;
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public String getName() {
             return jet.getName();
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public HazelcastInstance getHazelcastInstance() {
             return jet.getHazelcastInstance();
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public Cluster getCluster() {
             return jet.getCluster();
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public JetConfig getConfig() {
             return jet.getConfig();
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public Job newJob(@Nonnull Pipeline pipeline, @Nonnull JobConfig config) {
             return remember(jet.newJob(pipeline, updateJobConfig(config)));
         }
 
-        @Nonnull @Override
+        @Nonnull
+        @Override
         public Job newJob(@Nonnull DAG dag, @Nonnull JobConfig config) {
             return remember(jet.newJob(dag, updateJobConfig(config)));
         }
 
-        @Nonnull @Override
+        @Nonnull
```

Comment:
all those formatting changes look like done by mistake (maybe formatter not set or too much code formatted?)

Reply:
It was fixed after the review I think, I did not see this